python/bindings/openravepy_kinbody.cpp

```diff
@@ -654,25 +654,60 @@ PyJointControlInfo_RobotController::PyJointControlInfo_RobotController(const Joi
 {
     controllerType = jci.controllerType;
 
-    py::list _robotControllerAxisIndex;
+    py::list _robotControllerAxisProductCode, _robotControllerAxisIndex, _robotControllerAxisMult, _robotControllerAxisOffset;
```

Comment:
I think you are following the naming pattern of existing code, however we should not start local variable with ``_``.  please name to something else.

Reply:
renamed them into `axis{Index|Mult|Offset|ProductCode}`.